### ISSUE TITLE
get 54% top-1 acc on K600 val ? 

### ISSUE THREAD

**Issue by gooners1886** [facebookresearch/SlowFast#226]
I use the k600 model(https://dl.fbaipublicfiles.com/pyslowfast/model_zoo/ava/pretrain/SLOWFAST_64x2_R101_50_50.pkl)  with config file SLOWFAST_NLN_16x8_R101_50_50.yaml  to detect K600 val set. But I only get 
"top1_acc": "54.24", "top5_acc": "66.77"    which is far more lower than i expect.

detailed config is 
python  -u tools/run_net.py  \
--cfg configs/Kinetics/c2/SLOWFAST_NLN_16x8_R101_50_50.yaml  \
DATA.PATH_PREFIX /root/data/datasets/k600/val  \
DATA.PATH_TO_DATA_DIR data/k400   \
TEST.CHECKPOINT_FILE_PATH checkpoints_author/k600/SLOWFAST_64x2_R101_50_50.pkl   \
TRAIN.ENABLE False   \
TEST.CHECKPOINT_TYPE caffe2  \
NUM_GPUS 4 \
MODEL.NUM_CLASSES 600 \
TEST.BATCH_SIZE 8

Is there something i need to change?  
When will the action-name to label-index file be released?

**Comment by takatosp1** [comment]
Hi @gooners1886,

Thanks for playing with PySlowFast!
Seems SLOWFAST_NLN_16x8_R101_50_50 16x8 will build nonlocal slowfast R101 model with 16x8 inputs, where the weights are slowfast R101 without nonlocal with 64x2 inputs, you might want to modify the config to make the pretrain matches with the config :)
